tests/servers/threebot/test_package_manager.py

```diff
@@ -0,0 +1,103 @@
+import os
+from gevent import monkey
+from random import randint
+from jumpscale.loader import j
+from tests.base_tests import BaseTests
+
+monkey.patch_all(subprocess=False)
```

Comment:
I think monkey-patching should be done in tests module level, otherwise it would be defined many times (also, we need monkey patching to be enabled for all tests, not only threebot).

Reply:
I think the right place for this is [conftest.py](https://github.com/threefoldtech/js-sdk/blob/development/conftest.py) as it called while discovering the tests.